core/templates/components/forms/custom-forms-directives/image-uploader-modal.component.html

```diff
@@ -0,0 +1,221 @@
+<div class="oppia-image-uploader-modal-container">
+  <div class="modal-header">
+    <h3>{{ 'I18N_IMAGE_UPLOADER_MODAL_UPLOAD_HEADING' | translate:{ imageName: imageName } }}</h3>
+  </div>
+
+  <div class="modal-body oppia-image-uploader-container">
+    <div *ngIf="(isThumbnail() && imageNotUploaded())">
+      <strong tabindex="0">Please upload an SVG image.</strong>
+      <div class="alert alert-warning" tabindex="0">
+        <span>Here are some guidelines to follow:</span>
+        <ul>
+          <li>Uploaded image should have a transparent background.</li>
+        </ul>
+      </div>
+    </div>
+    <div class="oppia-image-uploader">
+      <div *ngIf="imageNotUploaded()">
+        <oppia-image-receiver (fileChanged)="onFileChanged($event)" [allowedImageFormats]="allowedImageFormats"
+          [maxImageSizeInKB]="maxImageSizeInKB">
+        </oppia-image-receiver>
+      </div>
+      <div class="oppia-form-error oppia-form-error-text" *ngIf="invalidImageWarningIsShown">
+        {{ 'I18N_IMAGE_UPLOAD_ERROR' | translate }}
+      </div>
+      <div *ngIf="(isThumbnail() && thumbnailImageDataUrl)">
+        <div [hidden]="!allowedBgColors">
+          <div [hidden]="!(allowedBgColors.length > 1)">Choose background color:</div>
+          <div [hidden]="!(allowedBgColors.length > 1)" class="oppia-thumbnail-colour-select">
+            <div *ngFor="let color of allowedBgColors" [ngStyle]="{'background': color}"
```

Comment:
Shouldn't this be inversed? `if allowedBgColors.length > 1` instead of `if !(allowedBgColors.length > 1)

Reply:
it's hidden so true means to not show it. replaced it with ngIf as once the allowedBgColors is passed we don't plan to change it so lets keep it removed from the DOM.